apps/models/app_types/shiny.py

```diff
@@ -23,16 +23,21 @@ class ShinyInstance(BaseAppInstance, SocialMixin, LogsEnabledMixin):
         ("public", "Public"),
         ("link", "Link"),
     )
+
+    volume = models.ForeignKey(
+        "VolumeInstance", blank=True, null=True, related_name="%(class)s", on_delete=models.CASCADE
```

Comment:
Just want to make sure, that this is the behavior that we want.

By setting `on_delete=models.CASCADE` when volume is deleted, this app will be deleted as well. It seems like we might not want that based on the recent changes with the subdomains

<img width="1263" alt="image" src="https://github.com/user-attachments/assets/c3043be0-ea73-45ff-a7fa-161f2e92e7cf">

[PR](https://github.com/ScilifelabDataCentre/stackn/pull/211/files#diff-1f62b2845f7575d4e30f0a07f7cb8f881419f886f178fdefa619a1f5293b24a7)

Reply:
True. I will change this. I think custom apps have the same cascade delete.